vulnerabilities/tests/test_data/openssf/openssf-data2.json-expected.json

```diff
@@ -0,0 +1,9 @@
+{
+  "aliases": ["MAL-2023-1077"],
```

Comment:
Should this be used as an alias? What other options we have?

cc @pombredanne

Reply:
@TG1999 yes, as an alias sounds fine